module_fcst_grid_comp.F90

```diff
@@ -745,19 +744,8 @@ subroutine fcst_initialize(fcst_comp, importState, exportState, clock, rc)
     endif
 ! if to write out restart at the end of forecast
     restart_endfcst = .false.
-    if ( ANY(frestart(:) == total_inttime) ) restart_endfcst = .true.
-! frestart only contains intermediate restart
-    do i=1,size(frestart)
-      if(frestart(i) == total_inttime) then
-        frestart(i) = 0
-        exit
-      endif
-    enddo
     if (mype == 0) print *,'frestart=',frestart(1:10)/3600, 'restart_endfcst=',restart_endfcst, &
       'total_inttime=',total_inttime
-! if there is restart writing during integration
-    intrm_rst         = 0
```

Comment:
declaration of `intrm_rst` can be deleted, if `intrm_rst` is not used anymore

Reply:
Done